tests/foreman/cli/test_ping.py

```diff
@@ -42,21 +42,13 @@ def test_positive_ping(target_sat, switch_user):
     """
     result = target_sat.execute(f"su - {'postgres' if switch_user else 'root'} -c 'hammer ping'")
     assert result.stderr[1].decode() == ''
-    assert result.status == 0
 
-    status_count = 0
-    ok_count = 0
-    # Exclude message from stdout for services candlepin_events and katello_events
-    result.stdout = [line for line in result.stdout.splitlines() if 'message' not in line]
+    # Filter lines containing "Status: "
+    statuses = [line for line in result.stdout.splitlines() if 'status' in line.lower()]
```

Comment:
There's a disconnect between the comment and the actual check. Personally, I think the comprehension is clear enough to not need a comment.

Reply:
I slightly changed the comment and also the filter expression. Now I filter "status:" (added the colon)